prince-of-versions/src/main/java/co/infinum/princeofversions/loaders/NetworkLoader.java

```diff
@@ -21,7 +21,7 @@
     /**
      * Default request timeout in seconds.
      */
-    public static final int DEFAULT_NETWORK_TIMEOUT_SECONDS = 60;
+    public static final int DEFAULT_NETWORK_TIMEOUT_SECONDS = 10;
```

Comment:
If we set this value to milliseconds than we change the interface of the class. All constructors take time in seconds, should we change that as well?